yt/frontends/cf_radial/data_structures.py

```diff
@@ -311,10 +311,10 @@ def _is_valid(cls, filename: str, *args, **kwargs) -> bool:
             with nc4_file.open_ds(keepweakref=True) as ds:
                 con = "Conventions"  # the attribute to check for file conventions
                 cons = ""  # the value of the Conventions attribute
-                for c in [con, con.lower()]:
+                for c in [con, con.lower(), "Sub_" + con.lower()]:
                     if hasattr(ds, c):
                         cons += getattr(ds, c)
-                is_cfrad = "CF/Radial" in cons
+                is_cfrad = "CF/Radial" in cons or "CF-Radial" in cons
```

Comment:
while your patch seems uncontroversial, I'm having trouble understand the existing implementation: why is `cons` constructed as a concatenated `str` instead of a `list[str]` ?

Reply:
I'll double check but I think it's because those attribute values are space-delimted strings. e.g., "CF-Radial instrument_parameters radar_parameters radar_calibration" is actually ```'CF-Radial instrument_parameters radar_parameters radar_calibration'``` so you have to search for a substring. I'll confirm that and add a comment in the code here (or update the code if I'm wrong).

Reply:
OK: confirmed just now, and pushed up a comment so that it's clearer in the future